spec/features/view_bookings_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.feature "Viewing past Bookings" do
+  before :each do
+    FactoryGirl.create(:booking_with_passengers)
+    FactoryGirl.create(:flight)
+    FactoryGirl.create(:plane)
+    FactoryGirl.create(:airline)
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.add_mock(:facebook, {:uid => "123456"})
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.